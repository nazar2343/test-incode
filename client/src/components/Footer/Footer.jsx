import React from "react";
import styles from "./Footer.module.css";

const Footer = () => {
  const year = new Date().getFullYear();

  return (
    <footer>
      <p className={styles.footer__text}>
        Copyright &copy; {year} Incode Group LLC. All rights reserved.
      </p>
    </footer>
  );
};

export default Footer;
