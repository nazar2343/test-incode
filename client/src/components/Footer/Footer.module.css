footer {
  margin-top: auto;
}

.footer__text {
  text-align: center;
  font-size: 14px;
}

@media screen and (max-width: 480px) {
  .footer__text {
    font-size: 12px;
  }
}
