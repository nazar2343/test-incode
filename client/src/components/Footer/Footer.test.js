import { render } from "@testing-library/react";
import Footer from "./Footer";

describe("<Footer /> component", () => {
  it("should match snapshot", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
