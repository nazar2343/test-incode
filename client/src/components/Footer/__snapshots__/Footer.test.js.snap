// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> component should match snapshot 1`] = `
<div>
  <footer>
    <p
      class="footer__text"
    >
      Copyright Â© 
      2022
       Incode Group LLC. All rights reserved.
    </p>
  </footer>
</div>
`;
